splashscreen.h

```diff
@@ -21,27 +25,25 @@ class Splashscreen {
 
         void draw_title_page()
         {
-            draw_sprite(background, 0, 0);
-
-            point_2d playPosition;
-            playPosition.x = 820;
-            playPosition.y = 620;
-
-            sprite_set_position(playButton, playPosition);
-            draw_sprite(playButton, 0, 0);
-
-            point_2d exitPosition;
-            exitPosition.x = 820;
-            exitPosition.y = 820;
+            // Draw buttons to screen
+            for (int i = 0; i < btns.size(); i++)
+            {
+                this->btns[i]->draw_button();
+            }
+            
+            // Draw background iamge to screen
+            draw_bitmap("bgnd", 0, 0);
 
-            sprite_set_position(exitButton, exitPosition);
-            draw_sprite(exitButton, 0, 0);
+            // Draw text on buttons
+            draw_text("play!", COLOR_BLANCHED_ALMOND, "btn_font", 80, btns[0]->x() - (btns[0]->centre_x()/2) + 5, btns[0]->y() - btns[0]->centre_y(), option_to_screen());
+            draw_text("options", COLOR_BLANCHED_ALMOND, "btn_font", 80, btns[1]->x() - (btns[1]->centre_x()/2) - 20, btns[1]->y() - btns[1]->centre_y(), option_to_screen());
+            draw_text("exit", COLOR_BLANCHED_ALMOND, "btn_font", 80, btns[2]->x() - (btns[2]->centre_x()/2) + 20, btns[2]->y() - btns[2]->centre_y(), option_to_screen());
         }
 
         void button_clicked(point_2d point)
```

Comment:
Consider refactoring to use the button classes, action function. 
(if not already intending to)